src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GermlineCNVNamingConstants.java

```diff
@@ -15,4 +15,6 @@
     public final static String BASELINE_COPY_NUMBER_TABLE_COLUMN = "BASELINE_COPY_NUMBER";
     public final static String SAMPLE_PREFIX = "SAMPLE_";
     public final static String INTERVAL_LIST_FILE_NAME = "interval_list.tsv";
+    public final static String DENOISED_COPY_RATIO_MEAN_FILE_NAME = "mu_denoised_copy_ratio_t.tsv";
+    public final static String DENOISED_COPY_RATIO_MEAN_TABLE_COLUMN = "DENOISED_COPY_RATIO_MEAN";
```

Comment:
Let's think carefully about what this column should be---we can discuss with @mwalker174 in person.